dashboard/src/components/ColoredCircle/ColoredCircle.tsx

```diff
@@ -40,13 +40,19 @@ const ColoredCircle = ({
   className,
   tooltipText,
 }: IColoredCircle): JSX.Element => {
+  const clearerCircles =
+    quantity == 0 &&
+    (tooltipText == 'global.failed' || tooltipText == 'global.inconclusive')
+      ? 'bg-opacity-20'
+      : '';
   return (
     <TooltipWrapper tooltipText={tooltipText}>
       <div
         className={cn(
           'inline-flex h-6 w-fit min-w-6 items-center justify-center rounded-full px-2 text-sm text-black',
           className,
           backgroundClassName,
+          clearerCircles,
```

Comment:
`cn` is a wrapper of the clsx library, you can do the contitional in another way 

```
{ ['bg-opacity-20']: quantity == 0 &&
    (tooltipText == 'global.failed' || tooltipText == 'global.inconclusive') }
```

but this way is also fine 

https://github.com/JedWatson/classnames